kong.yml

```diff
@@ -94,10 +94,19 @@
 ######
 ## PostgreSQL configuration
 # postgres:
-#    host: "127.0.0.1"
-#    port: 5432
-#    user: kong
-#    database: kong
+  #####
+  ## Connection host and port settings
+  # host: "127.0.0.1"
+  # port: 5432
+
+  ######
+  ## Name of the database used by Kong. Will be created if it does not exist.
+  # database: kong
+
+  #####
+  ## User authentication settings
+  # user: ""
+  # password: ""
```

Comment:
This makes the config file extremely verbose for very little benefit. I think everybody gets that `host` and `port` are `Connection host and port settings`...

Same for all the properties here. The *only* actual info is `Will be created if it does not exist.`, which can be documented at the `postgres` level of this file.